GOVERNANCE.md

```diff
@@ -26,6 +26,50 @@ Current administrators are
 - Sebastian Ehlert ([@awvwgk](https://github.com/awvwgk), awvwgk@disroot.org)
 - Laurence Kedward ([@lkedward](https://github.com/lkedward), laurence.kedward@bristol.ac.uk)
 
+### Decision-making process
+
+Occasionally, decisions need to be made about Fortran-lang projects, teams,
+communication, branding, funding, spending, public relations, and similar.
+This section describes the decision-making process for the Fortran-lang
+administrators team.
+
+* The administrators meet periodically and on an as-needed basis to discuss any
+  outstanding issues and make decisions on how to move forward. They will make
+  their best effort to promptly discuss and decide via email, audio calls,
+  and/or video calls.
+* Whenever appropriate, the administrators will request feedback from the
+  broader Fortran-lang community via GitHub and Fortran-lang Discourse. The
+  administrators will make their best effort to consider all feedback received
+  from the community while making a decision.
+* The administrators will make their best good-faith effort to act and make
+  decisions that are in the best long-term interest of the broader Fortran-lang
+  community, Fortran-lang projects, and the Fortran language as a whole.
+* Occasionally, there may not be unanimous agreement on any specific issue or
+  decision to be made. In such event, the administrators can vote to decide
+  how to move forward.
+* Any single administrators can call for a vote on a decision.
+* All administrators have equal voting power. For example, if there are five
+  administrators, each has the voting power of 20%.
+* The administrators will require 80% agreement to make any major decision.
+  This includes decisions such as:
+  - Adding or removing administrator team members;
+  - Changing the governance documents;
+  - Creating, removing, or transferring a Fortran-lang project (e.g. a GitHub
+    repository) or a service;
+  - Rebranding an existing Fortran-lang project;
+  - Applying for funding or contract work;
+  - Spending funds.
+  The purpose of the 80% requirement is to require a significant majority
+  agreement on a decision, while preventing any single member from blocking the
+  group from moving forward on that decision.
+* The administrators will require 51% agreement on all other decisions.
+* The administrator team will have at least five members. If an administrator
+  leaves the team for any reason, in which event the number of administrators
+  would become less than five, the remaining administrators will make their
+  best effort to replace the departing administrator.
+* No single administrator will act on behalf of Fortran-lang without first
+  discussing it with the administrators team.
```

Comment:
* Each administrator has a veto power and can veto a decision. This power should be only used as a last resort.

Reply:
That seems reasonable but how do we resolve the cases where +80% agreement has been reached but a veto has also been issued?